streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorStateManager.java

```diff
@@ -173,33 +173,24 @@ public void reinitializeStateStoresForPartitions(final Collection<TopicPartition
         return partitionsAndOffsets;
     }
 
-    List<ConsumerRecord<byte[], byte[]>> updateStandbyStates(final TopicPartition storePartition,
-                                                             final List<ConsumerRecord<byte[], byte[]>> records) {
+    void updateStandbyStates(final TopicPartition storePartition,
```

Comment:
Could you add in the description to explain changes of this function (i.e. throwing exception instead of returning remaining records)?